pkg/api/types/volume_types.go

```diff
@@ -16,11 +16,39 @@
 
 package types
 
-import "io"
+import (
+	"context"
+	"io"
 
-type VolumeLsCommandOptions struct {
+	"github.com/containerd/containerd"
+)
+
+type VolumeCommandOptions struct {
+	GlobalCommandOptions
 	// Writer is the output writer
 	Writer io.Writer
+}
+
+type VolumeCreateCommandOptions struct {
+	VolumeCommandOptions
```

Comment:
Should I use the anonymous field here?

Reply:
I think a standard filed could be better